app/services/import/persist-licence.service.js

```diff
@@ -156,6 +167,38 @@ async function _persistCompany (trx, updatedAt, company) {
     ])
 }
 
+async function _persistContact (trx, updatedAt, contact) {
+  return ContactModel.query(trx)
+    .insert({ ...contact, updatedAt })
+    .onConflict('externalId')
+    .merge([
+      'salutation',
+      'initials',
+      'firstName',
+      'lastName',
+      'updatedAt'
+    ])
+}
+
+async function _persistsCompanyContact (trx, updatedAt, companyContact) {
+  const { externalId, startDate, licenceRoleId } = companyContact
+
+  return db.raw(`
+    INSERT INTO public."company_contacts" (company_id, contact_id, licence_role_id, start_date, "default", created_at, updated_at)
```

Comment:
Issue with the on conflict not updating the existing record - it does not insert a new record so the conflict key must be working...